src/DependencyInjection/Configuration.php

```diff
@@ -49,6 +50,11 @@ public function getConfigTreeBuilder(): TreeBuilder
                                 ->info('When set to true, it will call normalize method with an extra groups parameter "groups" => [Searchable::NORMALIZATION_GROUP]')
```

Comment:
Should it be `serializer_groups`? 
```suggestion
                                ->info('When set to true, it will call normalize method with an extra groups parameter "serializer_groups" => [Searchable::NORMALIZATION_GROUP]')
```

If yes, I still see `groups` on `src/SearchableEntity.php` (but I don't know if they are related or not)

Reply:
> If yes, I still see groups on src/SearchableEntity.php (but I don't know if they are related or not)
That's because Symfony uses just `groups` both for normalization and denormalization